client/state/add-reducer.js

```diff
@@ -3,24 +3,41 @@
 /**
  * Internal Dependencies
  */
-
 import { APPLY_STORED_STATE } from 'state/action-types';
 import { getStateFromLocalStorage } from 'state/initial-state';
 
+const initializations = {};
+
+function normalizeKey( key ) {
+	return Array.isArray( key ) ? key.join( '.' ) : key;
```

Comment:
The key is always an array, so there doesn't need to be a conditional. (Could this module be converted to TypeScript?)

Reply:
Ah, I didn't realise it was always an array. As for the question, I guess I'm going to go and learn TypeScript now 😁

Reply:
Had to fix the tests, which were using strings as keys. Rewrote the file as Typescript.